cmd/proxy/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN yarn install --no-progress
 WORKDIR $GOPATH/src/github.com/gomods/athens
 
 ADD . .
-RUN cd cmd/proxy && buffalo build -s -o /bin/app
+RUN cd cmd/proxy && npm install && buffalo build -s -o /bin/app
```

Comment:
@marstr do you need the `npm install` here? I noticed that there is a `yarn install` above

Reply:
Derp, sorry about that. Fixed.